packages/vendure-plugin-shipmate/src/api/shipmate.service.ts

```diff
@@ -82,11 +82,14 @@ export class ShipmateService implements OnApplicationBootstrap {
   }
 
   /**
-   * Created a job in the job queue to send the order to Shipmate
+   * Creates a job in the job queue to send the order to Shipmate
    */
   async addJob(
     event: OrderStateTransitionEvent | OrderPlacedEvent
   ): Promise<void> {
+    if (!this.config.shouldSendOrder(event.ctx, event.order)) {
+      return;
```

Comment:
Can we add an info log statement here, including the order code? This way we can search in the logs if an order was deliberately not send to shipmate